edenai_apis/apis/google/google_audio_api.py

```
@@ -194,45 +197,68 @@ async def audio__atts(
         if is_gemini_model:
-            # Gemini TTS uses simple voice names like "Kore", "Puck", etc.
-            resolved_voice = voice or "Kore"
+            # Gemini TTS uses simple voice names like "kore", "puck", etc.
+            gemini_voices = [v for v in config["voices"] if "-" not in v]
+            if resolved_voice:
+                # Validate voice is a Gemini voice
+                if resolved_voice not in gemini_voices:
+                    raise ProviderException(
+                        f"Voice '{voice}' is not supported for Gemini model '{model}'. "
+                        f"Supported voices: {', '.join(gemini_voices)}"
+                    )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)